src/scripts/findConflicts.ts

```diff
@@ -0,0 +1,303 @@
+/**
+ * For running this script you should add a production.env in your config folder and run this:
+ *  NODE_ENV=production  ts-node src/scripts/findConflicts.ts
+ *  after running script a fill will be generated in ./src/scripts with including json
+ *  Then if you want to manually insert these donations to DB you should calculate valueUsd, ethUsd, userId,..
+ *  And after that you use below query:
+ */
+//
+// INSERT INTO public.donation ("transactionId", "transactionNetworkId", "toWalletAddress", "fromWalletAddress", currency,
+//   anonymous, amount, "userId", "projectId", "createdAt", "valueUsd", "valuEth", "priceEth", "priceUsd", "status")
+// values (transactionId, transactionNetworkId, toWalletAddress, fromWalletAddress, currency, anonymous, amount,
+//   userId, projectId, createdAt, valueUsd, valuEth, priceEth, priceUsd, status)
+
+import axios from 'axios';
+import { getEtherscanOrBlockScoutUrl, NETWORK_IDS } from '../provider';
+import { Container } from 'typedi';
+import * as TypeORM from 'typeorm';
+import { entities } from '../entities/entities';
+import { Donation } from '../entities/donation';
+import { Project } from '../entities/project';
+import { sleep } from '../utils/utils';
+import { writeFileSync } from 'fs';
+import { findTokenByNetworkAndSymbol } from '../utils/tokenUtils';
+import { isWalletAddressSmartContract } from '../utils/validators/projectValidator';
+
+const EXCLUDED_FROM_ADDRESSES = ['0x0000000000000000000000000000000000000000'];
+
+const smartContractAddresses: string[] = [];
+
+// 2021/12/24 16:00:00 GMT, beginning of round
+const beginTimestamp = 1640361600;
+
+// 2021/01/07 16:00:00 GMT, end of round
+const endTimestamp = 1641571200;
```

Comment:
Is the script reusable? Shouldn't we move this variables to env vars, so we only have to change them and rerun the script when required?

Reply:
Maybe for future rounds.

Reply:
You know when changing it for future round it's not needed to commit anything, I think we can merge it now, in the future we can change it
Thanks for your suggestion